### PR TITLE
Not sending metrics if developer build

### LINKED ISSUE #165: Don't send metrics from __DEV__ builds
App usage during development is skewing metrics results. We only want metrics from real usage.

### PR DESCRIPTION
**Pull request by arcturus**
closes #165

### CODE REVIEW

**Comment by arcturus** [comment]
@wilsonpage r?

@sgiles the nightly builds are dev or production?

**Comment by arcturus** [comment]
I mean @samgiles

File: lib/utils/metrics.js [hunk 1]
```diff
@@ -57,6 +57,10 @@ var exports = {};
       var post_url = 'https://www.google-analytics.com/batch';
       var event_string = formatEventString();
       debug('METRICS - event string:', event_string);
+      if (__DEV__) {
+        debug('METRICS - not sending to server cause developer build');
```
Comment (wilsonpage) [inline lib/utils/metrics.js]: NIT: `'METRICS - not sent for dev builds'`

Comment (arcturus) [inline lib/utils/metrics.js]: OOOOk

**Comment by wilsonpage** [comment]
@arcturus lint failure:

```
'__DEV__' is not defined  no-undef
```

Perhaps try `global.__DEV__`

**Comment by samgiles** [comment]
@arcturus Nightlies are 'release' builds (`gradle assembleRelease`), so production.

**Comment by arcturus** [comment]
@wilsonpage  Shame on me for not trying out  **DEV**

@samgiles Ok, we will need to tell @oteo and @isabelrios to build their own and don't take the prebuilds.

**Comment by arcturus** [comment]
Now tested on ios. @wilsonpage do you mind taking a look?

**Comment by samgiles** [comment]
He's on PTO - this looks good to me for now. r+

**Pull request merged**